source/user-manual/reference/centralized-configuration.rst

```diff
@@ -237,11 +237,12 @@ The following is an example of how a centralized configuration can be done.
 Precedence
 ----------
 
-It's important to understand which configuration file takes precedence between ``ossec.conf`` and ``agent.conf`` when central configuration is used. When central configuration is utilized, the local and the shared configuration are merged, however, the ``ossec.conf`` file is read before the shared ``agent.conf`` and the last configuration of any setting will overwrite the previous. Also, if a file path for a particular setting is set in both of the configuration files, both paths will be included in the final configuration.
+It's important to understand which configuration file takes precedence between ``ossec.conf`` and ``agent.conf`` when central configuration is used.
+When central configuration is utilized, the local and the shared configuration are merged, however, the ``ossec.conf`` file is read before the shared ``agent.conf`` and the last configuration of any setting will overwrite the previous. Also, if a file path for a particular setting is set in both of the configuration files, both paths will be included in the final configuration.
 
```

Comment:
Here I would make it clear that there are configurations that are merged and others that are overwritten. 

I would then explain in which cases it would be added and in which cases it would be overwritten.

Finally, I would give as an example the cases mentioned below